pulp-in-one-container.md

```diff
@@ -25,7 +23,7 @@ $ mkdir settings pulp_storage pgsql containers
 $ echo "CONTENT_ORIGIN='http://$(hostname):8080'
 ANSIBLE_API_HOSTNAME='http://$(hostname):8080'
 ANSIBLE_CONTENT_HOSTNAME='http://$(hostname):8080/pulp/content'
-TOKEN_AUTH_DISABLED=True" >> settings/settings.py
+CACHE_ENABLED=True" >> settings/settings.py
 ```
 
 ## With SELinux
```

Comment:
Does it make sense to add a note about `sudo getenforce`? So, users will know which command to run?